workflows/prognostic_run_diags/tests/test_registry.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import xarray as xr
+import pytest
+from typing import Sequence, Tuple
+
+from fv3net.diagnostics.prognostic_run.registry import Registry
+
+
+@pytest.fixture()
```

Comment:
Would prefer a `_get_registry()` function to this fixture to make it easier to parameterize in the future without leading the fixture-house-of-cards phenomena which has plagued us in the past.